README.md

```diff
@@ -70,26 +70,113 @@ or you can directly download [sources](https://github.com/Zenika/immutadot/relea
 
 ## Usage
 
-in browser:
+ES modules:
 
 ```js
-  import { push } from 'immutadot'
+import { set } from 'immutadot'
+```
+
+CommonJS:  
 
-  push({ nested: { prop: [1, 2] } }, 'nested.prop', 3, 4)
-  // → { nested: { prop: [1, 2, 3, 4] } }
+```js
+const { set } = require('immutadot')
 ```
 
-in node:
+### Example
+
+Object used in the following example:
 
 ```js
-  const { push } = require('immutadot')
+const animals = {
+  weasel: [
+    {
+      vernacularName: 'badger',
```

Comment:
You forgot to capitalize the vernacular names in the input.
It may be the occasion to quickly show the slice notation and immutadot-lodash :
```js
import { capitalize } from 'immutadot-lodash'
const newAnimals = capitalize(animals, 'weasel[:].vernacularName')
```
vs
```js
import { capitalize } from 'lodash'
const newAnimals = {
  ...animals,
  weasels: animals.weasels.map(weasel => {
    return {
      ...weasel,
      vernacularName: capitalize(weasel.vernacularName),
    }
  }),
}
```